dagger/logger/logger.go

```diff
@@ -0,0 +1,65 @@
+package logger
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+)
+
+const (
+	defaultOutputDir = ".daggers"
+	defaultFileName  = "dagger.log"
+)
+
+var (
+	_ io.Writer = new(Logger)
+	_ io.Closer = new(Logger)
+)
+
+// Logger is a simple interface wrapper around the standard io.Writer interface.
+type Logger struct {
+	w io.Writer
+}
+
+// NewLogger returns simple logger that writes logs to a log file under default output directory(.daggers). If verbose
+// is true, logs will be shown in os.Stdout at the sametime.
+func NewLogger(verbose bool) (Logger, error) {
+	logger := Logger{}
+
+	wd, err := os.Getwd()
+	if err != nil {
+		return logger, err
+	}
+
+	if err := os.MkdirAll(filepath.Join(wd, defaultOutputDir), os.ModePerm); err != nil {
+		return logger, err
+	}
+
+	filePath := filepath.Join(wd, defaultOutputDir, defaultFileName)
+
+	file, err := os.OpenFile(filePath, os.O_APPEND|os.O_CREATE|os.O_WRONLY, os.ModePerm)
```

Comment:
How about truncating the file each time? Otherwise this file will infinitely grow when running locally.

Reply:
Will add follow up PR for easier decision - want to merge this!